src/main/resources/db-scripts/clickhouse/clickhouse_migration.sql

```diff
@@ -1,35 +1,45 @@
 
--- Genomic Event Mutation Data
-Insert into genomic_event_mutation
-SELECT  concat(cs.cancer_study_identifier, '_', sample.stable_id) as sample_unique_id,
-        me.protein_change                                         as variant,
-        gene.hugo_gene_symbol                                     as hugo_gene_symbol,
-        gp.stable_id                                              as gene_panel_stable_id,
-        cs.cancer_study_identifier                                as cancer_study_identifier,
-        g.stable_id                                               as genetic_profile_stable_id,
-        me.mutation_type                                          as mutation_type,
-        mutation.mutation_status                                  as mutation_status,
-        'NA'                                                      as driver_filter,
-        'NA'                                                      as drivet_tiers_filter
-FROM mutation
-         INNER JOIN mutation_event as me on mutation.mutation_event_id = me.mutation_event_id
-         INNER JOIN sample_profile sp
-                    on mutation.sample_id = sp.sample_id and mutation.genetic_profile_id = sp.genetic_profile_id
-         LEFT JOIN gene_panel gp on sp.panel_id = gp.internal_id
-         LEFT JOIN genetic_profile g on sp.genetic_profile_id = g.genetic_profile_id
-         INNER JOIN cancer_study cs on g.cancer_study_id = cs.cancer_study_id
-         INNER JOIN sample on mutation.sample_id = sample.internal_id
-         LEFT JOIN gene on mutation.entrez_gene_id = gene.entrez_gene_id;
+INSERT INTO sample_to_gene_panel
+select
+    concat(cs.cancer_study_identifier, '_', sample.stable_id) as sample_unique_id,
+    genetic_alteration_type as alteration_type,
+    ifnull(gene_panel.stable_id, 'WES') as gene_panel_id,
```

Comment:
lets put a comment explaining this since it's importanat and a lot of people won't know what WES is.  (whole exome sequencing?).